restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java

```diff
@@ -6,7 +6,13 @@
 import com.google.common.io.CharStreams;
 import com.samskivert.mustache.Template;
 import restx.common.processor.RestxAbstractProcessor;
-import restx.factory.*;
+import restx.factory.Alternative;
+import restx.factory.Component;
+import restx.factory.Machine;
+import restx.factory.Module;
+import restx.factory.NamedComponent;
+import restx.factory.Provides;
+import restx.factory.When;
```

Comment:
I don't see how this make the module compilable with java 11? Maybe you omitted something in your commit?

Reply:
Sorry, I didn't provide details about the error.

When executing `mvn verify` with jdk11, the following errors happen during `restx-factory-annotation-processor` compilaton.

```
INFO] 
[INFO] ------------< io.restx:restx-factory-annotation-processor >-------------
[INFO] Building restx-factory-annotation-processor 0.36-SNAPSHOT         [4/41]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-maven) @ restx-factory-annotation-processor ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ restx-factory-annotation-processor ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 6 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.7.0:compile (default-compile) @ restx-factory-annotation-processor ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 2 source files to /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java:[178,40] reference to Module is ambiguous
  both class restx.factory.Module in restx.factory and class java.lang.Module in java.lang match
[ERROR] /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java:[193,51] reference to Module is ambiguous
  both class restx.factory.Module in restx.factory and class java.lang.Module in java.lang match
[ERROR] /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java:[71,69] reference to Module is ambiguous
  both class restx.factory.Module in restx.factory and class java.lang.Module in java.lang match
[ERROR] /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java:[79,17] reference to Module is ambiguous
  both class restx.factory.Module in restx.factory and class java.lang.Module in java.lang match
[ERROR] /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java:[79,53] reference to Module is ambiguous
  both class restx.factory.Module in restx.factory and class java.lang.Module in java.lang match
[INFO] 5 errors 
[INFO] -------------------------------------------------------------
```

Removing the wild card and specifying each classes, especially `restx.factory.Module`, remove the ambiguous reference with `java.lang.Module` introduced in jdk9.

Reply:
Ok, got it, thanks.